docs/conf.py

```diff
@@ -12,18 +12,23 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
-import sys
-sys.path.insert(0, os.path.abspath('../'))
-sys.path.insert(0, os.path.abspath('.'))
-sys.path.insert(0, os.path.abspath('./'))
+import datetime
+
+from ansible_runner.__main__ import VERSION
 
 
 # -- Project information -----------------------------------------------------
 
+def _get_version():
+    version_parts = VERSION.split('.', 3)[:3]
+
+    return '.'.join(version_parts)
+
+
 project = 'ansible-runner'
-copyright = '2018, Red Hat Ansible'
-author = 'Red Hat Ansible'
+copyright = f'2018-{datetime.datetime.today().year}, Red Hat, Inc'
```

Comment:
Usually, the copyright year shows the start of the content creation and the follow-up year updates demonstrate when the updates have been made. The current year is not always the year of an update, just rebuild. So I'm not sure if it makes sense to use it.

Besides, according to the Linux Foundation, it's reasonable to drop this completely since the authors preserve the copyright regardless of the presence of the copyright notice which is hard to track/update over time: https://linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/.

P.S. If you reorder the `author` and `copyright` vars, you could use the former in the f-string too.

Reply:
Copyright stuff is really fuzzy. I mainly was trying to make the site not look stale since old copyright years can (incorrectly) be interpreted as "this site hasn't been updated in a while".

I read through Red Hat's guidance on copyright notices and it sounded like the year should be the year any content was created. I figured so long as we are adding _something_ new in a given calendar year, the dates will be accurate. But that is a bit fuzzy, for sure.

Reply:
> P.S. If you reorder the author and copyright vars, you could use the former in the f-string too.

The minor annoyance I ran into with that is the trailing `.` is added at the end of the copyright notice. I don't want the `.` to exists in the copyright notice since it would render as `2018-2021 Red Hat, Inc..`, but I _do_ want it in the `author` string.